target-platforms/default.tpd

```diff
@@ -0,0 +1,77 @@
+target "Default (clean build) Target Platform"
+
+// include "testing/4.32-2024-06.tpd" with source requirements environment JavaSE-17
+
+with source requirements
+environment JavaSE-17
+
+location "https://download.eclipse.org/eclipse/updates/4.32" {
```

Comment:
We should use the repository that has the timestamp in the URL, if available, to increase reproducability of the build.
However, shouldn't we better use the Eclipse "release" here, rather than the "update"?